app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    our_list = []
+    for human in people:
+        people = Person(human["name"], human["age"])
```

Comment:
The variable `people` here shadows the input parameter `people` of the function `create_person_list`. Consider renaming this variable to avoid confusion and potential bugs.